tooling/nargo_cli/tests/stdlib-props.rs

```diff
@@ -259,23 +259,16 @@ fn fuzz_sha512_equivalence() {
 fn fuzz_poseidon2_equivalence() {
     use bn254_blackbox_solver::poseidon_hash;
 
-    for max_len in [0, 1, 3, 4, 511, 512] {
+    for max_len in [0, 1, 100] {
```

Comment:
Is there a reason for reducing the number of lengths we test?

Reply:
It took ~35 seconds to test with the larger inputs. Since I just copied them from how you added it to one of the SHAs, I thought the 512 boundary is probably not that relevant to this one. Do you have any preference in this case?

Reply:
Added back 3 and 4 - there is some logic that kicks in after absorbing 3 inputs, maybe that's a good place to probe.